### ISSUE TITLE
Extension incompatible with jupyterlab>=1.1

### ISSUE THREAD

**Issue by zaneselvans** [pbugnion/jupyterlab-sql#98]
This extension seems to be incompatible with the most recent releases of jupyterlab (>=1.1.0):
```
$ jupyter lab --version
1.1.1
$ jupyter labextension update jupyterlab-sql
No compatible version found for jupyterlab-sql!
$ jupyter labextension list
JupyterLab v1.1.1
Known labextensions:
   app dir: /home/zane/miniconda3/envs/pudl/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v1.0.2  enabled  OK
        @jupyterlab/toc v1.0.1  enabled  OK
        jupyterlab-sql v0.3.0  enabled   X
        jupyterlab_vim v0.11.0  enabled  OK
        qgrid v1.1.1 disabled OK

   The following extension are outdated:
        jupyterlab-sql
```
Not sure if this is a true incompatibility, or if a list of compatible versions needs to be updated.

It's a great extension for making working with databases more convenient. Thank you for your efforts!

**Comment by helloandrey** [comment]
Same issue on:
```
$ jupyter lab --version
1.1.2
```

**Comment by euanmacinnes** [comment]
Getting the same issue still with JL 1.1.3

**Comment by pbugnion** [comment]
Thanks for reporting, and apologies for the silence.

This is now fixed by PR #99. I've released 0.3.1-rc4 that supports both JupyterLab 1.0.x and 1.1.x.

To use the release candidate:

```
pip install jupyterlab_sql==0.3.1-rc4
jupyter serverextension enable jupyterlab_sql --py --sys-prefix
jupyter lab build
```

**Comment by pbugnion** [comment]
Now available in jupyterlab_sql 0.3.1. I'm closing this, but feel free to re-open if you have further issues.
